src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -121,6 +121,34 @@
   private static final ConfigDef.Validator DELIVERY_GUARANTEE_VALIDATOR =
       new DeliveryGuaranteeValidator();
 
+  // For error handling
+  public static final String ERROR_GROUP = "ERRORS";
+  public static final String ERRORS_TOLERANCE_CONFIG = "errors.tolerance";
+  public static final String ERRORS_TOLERANCE_DISPLAY = "Error Tolerance";
+  public static final ErrorTolerance ERRORS_TOLERANCE_DEFAULT = ErrorTolerance.NONE;
+  public static final String ERRORS_TOLERANCE_DOC =
+      "Behavior for tolerating errors during Sink connector's operation. 'NONE' is set as default"
+          + " and denotes that it will be fail fast. i.e any error will result in an immediate task"
+          + " failure. 'ALL'  skips over problematic records.";
+
+  public static final String ERRORS_LOG_ENABLE_CONFIG = "errors.log.enable";
+  public static final String ERRORS_LOG_ENABLE_DISPLAY = "Log Errors";
+  public static final boolean ERRORS_LOG_ENABLE_DEFAULT = false;
+  public static final String ERRORS_LOG_ENABLE_DOC =
+      "If true, write/log each error along with details of the failed operation and record"
+          + " properties to the Connect log. Default is 'false', so that only errors that are not"
+          + " tolerated are reported.";
+
+  public static final String ERRORS_DEAD_LETTER_QUEUE_TOPIC_NAME_CONFIG =
+      "errors.deadletterqueue.topic.name";
+  public static final String ERRORS_DEAD_LETTER_QUEUE_TOPIC_NAME_DISPLAY =
+      "Send error records to the dead letter queue(DLQ)";
```

Comment:
nit, capitalize each letter and add a space between "queue" and ("DLQ)"

Reply:
Will do! thanks